R/animint.R

```diff
@@ -856,7 +858,9 @@ animint2dir <- function(plot.list, out.dir = tempfile(),
         update.vars <- L$mapping[is.ss | is.cs]
         has.var <- update.vars %in% names(L$data)
         if(!all(has.var)){
-          print(update.vars[!has.var])
+          print(L)
+          print(list(problem.aes=update.vars[!has.var],
+                     data.variables=names(L$data)))
           stop("data does not have interactive variables")
```

Comment:
this error message is now a bit more user-friendly: `print(L)` should help the user identify which layer caused the error in a complex multi-layer plot.